python/cuml/cuml/tests/test_incremental_pca.py

```diff
@@ -50,48 +53,53 @@ def test_fit(
     sparse_format,
     batch_size_divider,
     whiten,
+    device,
 ):
-
-    if sparse_format == "csc":
-        pytest.skip(
-            "cupyx.scipy.sparse.csc.csc_matrix does not support"
-            " indexing as of cupy 7.6.0"
+    with using_device_type(device):
+
+        if sparse_format == "csc":
+            pytest.skip(
+                "cupyx.scipy.sparse.csc.csc_matrix does not support"
+                " indexing as of cupy 7.6.0"
+            )
```

Comment:
Can't we remove the cases where `sparse_format == "csc"`from the parametrisation? And then also remove this skipping. One less bit of noise in the test output log :D

At least it sounds like that indexing has been remove in an old cupy version -> it ain't coming back any time soon